src/DqtApi/Program.cs

```diff
@@ -65,6 +73,11 @@ public static void Main(string[] args)
             services
                 .AddMvc(options =>
                 {
+                    // Replace the standard BodyModelBinderProvider with our HybridBodyModelBinderProvider decorator
+                    var bodyModelBinderProvider = options.ModelBinderProviders.OfType<BodyModelBinderProvider>().Single();
+                    var hybridBodyModelBinderProvider = new HybridBodyModelBinderProvider(bodyModelBinderProvider);
+                    options.ModelBinderProviders[options.ModelBinderProviders.IndexOf(bodyModelBinderProvider)] = hybridBodyModelBinderProvider;
```

Comment:
move to extension? instead of littering with comments.

```csharp
options.ModelBinderProviders.ReplaceStandardBodyModelBinderProvider()
```

Reply:
Yep - that's definitely cleaner